go/obscuronode/enclave/enclave.go

```diff
@@ -171,7 +173,7 @@ func (e *enclaveImpl) start(block types.Block) {
 		// A new winner was found after gossiping. Start speculatively executing incoming transactions to already have a rollup ready when the next round starts.
 		case winnerRollup := <-e.roundWinnerCh:
 			hash := winnerRollup.Hash()
-			env.header = obscurocore.NewHeader(&hash, winnerRollup.Header.Number+1, e.config.HostID)
+			env.header = obscurocore.NewHeader(&hash, winnerRollup.Header.Number.Uint64()+1, e.config.HostID)
```

Comment:
It's a bit annoying to have to get the `.Uint64()` all over the place. Would it make sense to make some of these height fields etc. a big int to match the header type?

Reply:
I know.. I tried somthing like that but the changes were quite extensive, so I gave up